p2p/src/network/scheduler/p2p_network_scheduler_reducer.rs

```diff
@@ -374,7 +374,6 @@ impl P2pNetworkSchedulerState {
                 let state: &P2pState = state.substate()?;
 
                 let peer_with_state = state.peer_with_connection(*addr);
-                dispatcher.push(P2pNetworkSchedulerAction::Prune { addr: *addr });
```

Comment:
Why is `P2pNetworkSchedulerAction::Prune` dispatch removed, it seems to be a reason why p2p tests are failing

Reply:
This is postponed. We remove the connection after mio responds that the connection is closed, not immediately.